src/GraphQL.Tests/Bugs/Issue2387_OverrideBuiltInScalars.Alt.cs

```diff
@@ -0,0 +1,259 @@
+using System.Threading.Tasks;
+using GraphQL.Language.AST;
+using GraphQL.SystemTextJson;
+using GraphQL.Types;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/issues/2387
+    public class Issue2387_OverrideBuiltInScalars_Alt : QueryTestBase<Issue2387_OverrideBuiltInScalars_Alt.MySchema>
```

Comment:
```suggestion
    // https://github.com/graphql-dotnet/graphql-dotnet/issues/2392
    public class Issue2392_OverrideBuiltInScalars_Alt : QueryTestBase<Issue2392_OverrideBuiltInScalars_Alt.MySchema>
```
???  either way is fine with me.

Reply:
Done